epilepsy12/common_view_functions/sanction_user_access.py

```diff
@@ -29,3 +22,35 @@ def logged_in_user_may_access_this_organisation(user, organisation_requested):
     else:
         # user is not active or email confirmed
         return False
+
+
+def organisation_white_list_for_user(epilepsy12_user):
```

Comment:
Do we need this helper method rather than using `epilepsy12_user.employers.all()` directly? At least for the check in `user_may_view_this_organisation` if I do

```python
organisation_requested in user.employers.all()
```

it still seems to work

Reply:
I think the `epilepsy12_user.employers.all()` with return all organisations that the user is affiliated with. But I had understood that the lead clinician needed access to all the organisations in the governing Trust or Local Health Board. The helper function runs a query to return all the sibling organisations of all the parents the user is affiliated, not just the organisation itself. The function is run in two places (in the view to populate the dropdown and here in the decorator), which is why I abstracted it out into its own function.